spec/puppet-syntax/manifests_spec.rb

```diff
@@ -108,19 +123,44 @@
         end
       end
     end
+
+    if Puppet.version.to_i == 4
```

Comment:
The reason I did that is because I'm trying to match `>= 4.0.0` and as far as I know you can't in a case statement.

I could try to match `4.0..4.99` maybe? Thoughts?